iree/base/internal/BUILD

```diff
@@ -151,12 +155,32 @@ cc_test(
 
 cc_library(
     name = "flags",
-    srcs = ["flags.cc"],
+    srcs = ["flags.c"],
     hdrs = ["flags.h"],
     deps = [
+        ":internal",
         "//iree/base:api",
-        "@com_google_absl//absl/flags:parse",
+        "//iree/base:tracing",
+    ],
+)
+
+cc_binary(
```

Comment:
> In cmake it's `iree_base_internal_flags_demo` - but maybe that's because it's a test - I'll try changing it to cc_binary instead. I was using cc_test because I want the binary to be run under CI independently (as if a user was running it on the command line, not our test) - if I make it a cc_binary it won't be run. I'd rather have it work, though :)

If you want to run a binary as a test you can use `run_binary_test` (https://github.com/google/iree/blob/main/build_tools/bazel/run_binary_test.bzl)

Reply:
Cool - hadn't seen that! I think the lit coverage here is probably fine now.